include/sensors.h

```diff
@@ -149,12 +149,12 @@ class Sensors : public ParamListenerInterface
     float diff_pressure_velocity = 0;
     float diff_pressure = 0;
     float diff_pressure_temp = 0;
-    bool diff_pressure_valid = false;
+  //  bool diff_pressure_valid = false;
```

Comment:
Could you remove this line instead of just comment it out?

Reply:
updated.